docs/lib/examples/ButtonDropdownMulti.svelte

```diff
@@ -0,0 +1,21 @@
+<script>
+	import { ButtonDropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'sveltestrap';
+
+	export let color;
+	export let text;
+
+	let isOpen = false;
```

Comment:
I'm seeing an issue with the docs - when one ButtonDropdown is opened, they all open:

<img width="1025" alt="Screen Shot 2019-06-07 at 8 41 51 PM" src="https://user-images.githubusercontent.com/405608/59141616-4bec7280-8965-11e9-80de-7ec1d2a3c2e3.png">

The code looks correct though, any idea why this would be shared across instances?

Reply:
Yeah, it's a problem with the shared state. I don't know of a way to do it correctly without passing a bunch of callbacks around.

I noted the problem in the readme

Reply:
Okay, I'll take a look if can resolve before releasing new version.